runtime/Cargo.toml

```diff
@@ -20,6 +20,8 @@ log = { version = "0.4.14", default-features = false }
 scale-info = { version = "1.0.0", default-features = false, features = ["derive"] }
 serde = { version = "1.0.101", optional = true, features = ["derive"] }
 smallvec = "1.6.1"
+# This getrandom package was having issues been compiled to wasm in a downstream dependency, that's why this override is here
+getrandom = { version = "0.2", features = ["js"] }
```

Comment:
This should never be in a runtime, `getrandom` doesn't compile to WASM that Substrate does afaik.

Reply:
I got an exact error that getrandom was not being compiled to wasm, getrandom can get compiled to wasm but the js feature needs to be enabled